### PR TITLE
Features

### PR DESCRIPTION
**Pull request by alidurrani1**
Solved minor issues with docker as well.
Structured the whole code.
removed unnecessary code.
mail implementation.
celery scheduler.

### CODE REVIEW

File: app/__init__.py [hunk 1]
```diff
@@ -1,17 +1,22 @@
-from flask_sqlalchemy import SQLAlchemy
+from celery.schedules import crontab
 from flask_wtf.csrf import CSRFProtect
+from flask_sqlalchemy import SQLAlchemy
 from flask import Flask
 from app.make_celery import *
-from  werkzeug.security import generate_password_hash, check_password_hash
+from werkzeug.security import generate_password_hash, check_password_hash
```
Comment (maimoonaiqbal94) [inline app/__init__.py]: pep8, organise imports alphabetically.

File: app/__init__.py [hunk 2]
```diff
@@ -23,9 +28,11 @@
 app.config.update(CELERYBEAT_SCHEDULE={
     'say-every-5-seconds': {
         'task': 'store',
-        "schedule": 86400.00,
-         }
+        "schedule": crontab(hour='*/24'),
+    }
 })
+session_token = ''
```
Comment (maimoonaiqbal94) [inline app/__init__.py]: what does this mean? why are you declaring it with empty strings?

**Review by maimoonaiqbal94 — COMMENTED** [review]
Please remove pyc files.

File: app/api.py [hunk 3]
```diff
@@ -1,12 +1,22 @@
...
 from app import db
 
 import json
 import urllib
 import requests
+from marshmallow import Schema, fields
+
+
+# Schema For API
+class CarSchema(Schema):
```
Comment (maimoonaiqbal94) [inline app/api.py]: create a separate file for schemas, and import it from there.

File: requirements.txt [hunk 4]
```diff
@@ -1,35 +1,26 @@
...
+fasteners==0.14.1
+Flask==2.2.2
+Flask-Mail==0.9.1
+Flask-SQLAlchemy==3.0.2
 Flask-WTF==1.0.1
-future==0.18.2
-greenlet==2.0.1
 Jinja2==3.1.2
-Mako==1.1.3
+jsonschema==3.2.0
```
Comment (maimoonaiqbal94) [inline requirements.txt:8]: Please make sure to include those requirements which are used in the project.

File: app/api.py (hunk 3 above)
Comment (alidurrani1) [reply app/api.py]: got it and done.

File: app/__init__.py (hunk 2 above)
Comment (alidurrani1) [reply app/__init__.py]: initialized that string because it is used in views.py file in conditional statements.

File: app/__init__.py (hunk 1 above)
Comment (alidurrani1) [reply app/__init__.py]: got it  and done.

File: requirements.txt (hunk 4 above)
Comment (alidurrani1) [reply requirements.txt:8]: Some how my pip3 freeze command includes unnecessary requirements working on it.

**Pull request merged**